drivers/frequency/hmc7044/hmc7044.h

```diff
@@ -92,6 +93,7 @@ struct hmc7044_dev {
 
 struct hmc7044_init_param {
 	spi_init_param	*spi_init;
+	bool		is_hmc7043;
```

Comment:
Both 7043 and 7044 got device id registers (0x78 0x79 0x7A). It is the same device id? If not, this member can be dropped and get the device type from the registers